gaming/minecraft/en.md

```diff
@@ -33,7 +44,7 @@ Type=Application
 Name=Minecraft
 Comment=Minecraft Launcher
 Icon=minecraft
-Exec=java -jar Minecraft.jar
+Exec=/opt/minecraft/./minecraft-launcher.sh
```

Comment:
`./` isn't needed. Can just be `/opt/minecraft/minecraft-launcher.sh`

Reply:
Good to know, will fix 👍